_includes/integrations/react-tech-ref.md

```diff
@@ -13,51 +13,104 @@
   - [`initialValue`](#initialvalue)
   - [`inline`](#inline)
   - [`onEditorChange`](#oneditorchange)
-  - [`outputFormat`](#outputformat)
+  - [`outputFormat` (*deprecated*)](#outputformat)
   - [`plugins`](#plugins)
+  - [`scriptLoading`](#scriptloading)
   - [`tagName`](#tagname)
   - [`textareaName`](#textareaname)
   - [`toolbar`](#toolbar)
   - [`tinymceScriptSrc`](#tinymcescriptsrc)
   - [`value`](#value)
 
+- [Using the TinyMCE React component as a uncontrolled component](#usingthetinymcereactcomponentasauncontrolledcomponent)
 - [Using the TinyMCE React component as a controlled component](#usingthetinymcereactcomponentasacontrolledcomponent)
 - [Event binding](#eventbinding)
 
-### Installing the TinyMCE React integration using NPM
+### Installing the TinyMCE React integration using NPM or Yarn
 
 To install the `tinymce-react` package and save it to your `package.json`.
 
 ```sh
 $ npm install --save @tinymce/tinymce-react
 ```
 
-### Configuring the editor
-
-The editor accepts the following properties:
+or with Yarn
 
-```jsx
-<Editor
-  apiKey='your-api-key'
-  cloudChannel='{{site.productmajorversion}}-stable'
-  disabled={false}
-  id='uuid'
-  init= {% raw %}{{{% endraw %}  {% raw %}}}{% endraw %}
-  initialValue=''
-  inline={false}
-  onEditorChange={}
-  plugins=''
-  tagName='div'
-  textareaName=''
-  toolbar=''
-  value=''
-/>
+```sh
+$ yarn add @tinymce/tinymce-react
 ```
 
-None of the configuration properties are **required** for `tinymce-react` to work. Specify a {{site.cloudname}} API key using `apiKey` to remove the `This domain is not registered...` warning message.
+### Configuring the editor
+
+The following are the most common properties grouped by category.
+
+#### Configuring editor source
+The integration will load {{site.productname}} from these sources in this priority:
+1. If the global `tinymce` is present on the page nothing additional will be loaded and it will be used.
+2. If the `tinymceScriptSrc` property is provided then a script tag will be added to load {{site.productname}} from the given URL.
+3. Finally if none of the above conditions apply then a script tag will be added to load {{site.productname}} from {{site.cloudname}}.
+
+These settings are used to configure this behavior further:
+
+<dl>
+<dt><a href="#apikey"><code>apiKey</code></a></dt>
+<dd>The {{site.cloudname}} API key. When loading from {{site.cloudname}} specify this to remove the &quot;This domain is not registered...&quot; warning message.</dd>
+<dt><a href="#cloudchannel"><code>cloudChannel</code></a></dt>
+<dd>The build of {{site.productname}} used when loading from {{site.cloudname}}.</dd>
```

Comment:
```suggestion
<dd>The channel of {{site.productname}} used when loading from {{site.cloudname}}.</dd>
```
Build isn't really accurate (it's more the release) and it's called channel elsewhere, so this keeps it consistent.

Reply:
done